template/koa-rest-api/.gantry/common.yml

```
@@ -1,11 +1,12 @@
 prodAccountId: '<%- prodAwsAccountId %>'
 
-image: '{{values "prodAccountId"}}.dkr.ecr.ap-southeast-2.amazonaws.com/{{values "service"}}:{{.BuildID}}'
+image: '{{values "prodAccountId"}}.dkr.ecr.{{values "region"}}.amazonaws.com/{{values "service"}}:{{.BuildID}}'
```

should this be

```suggestion
image: '{{values "prodAccountId"}}.dkr.ecr.{{.Region}}.amazonaws.com/{{values "service"}}:{{.BuildID}}'
```

or

```suggestion
image: '{{values "prodAccountId"}}.dkr.ecr.<%- region %>.amazonaws.com/{{values "service"}}:{{.BuildID}}'
```

---

2nd is clearer imo,  and more straightforward for people to change if its a cross region service